pages/index.jsx

```diff
@@ -49,3 +60,4 @@ const Home = () => {
 }
```

Comment:
So instead of commenting out other work (even if it's out of date) you should add what you have by changing 
```
return <HeroGrid content={content} />
```
to 
```
    return (
        <div>
            <HeroGrid content={content} />
            <AboutComponent />
        </div>
    )
```